### PR TITLE
Dashboard redesign

### PR DESCRIPTION
**Pull request by liviavinci**
screen shot 2018-03-03 at 6 28 50 pm

### CODE REVIEW

**Comment by liviavinci** [comment]
Please note that the code within TopActons/index.js are old code and they are temporary. Will change components to material and reorg the code in a later PR.

File: src/components/NavBar/components/NavEventsButtons/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    return (
+      <div>{
+        this.props.buttons.map((entry, index) => (
+          <Button
+            key={entry.text}
+            data-index={index}
+            onClick={this.onBtnClicked}
+            className={classNames(classes.navEventsButton, index === _.toNumber(this.props.tabIndex) ? 'selected' : '')}
+          >
+            <FormattedMessage id={`dashboard.${entry.text}`} />
```
Comment (dwalintukan) [inline src/components/NavBar/components/NavEventsButtons/index.js]: hm weird, Henry said you have to your `intl.format` for string templates. guess not 🤷‍♂️

File: src/components/NavBar/index.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            }, {
+              text: 'Finalize',
+            }, {
+              text: 'Withdraw',
+            }]}
+          />
+          <div className={classes.navBarRightWrapper}>
+            <Button className={classes.navBarWalletButton}>
+              <AccountBalanceWalletIcon fontSize className={classes.navBarWalletIcon} />
+              {this.getTotalQTUM()} QTUM / {this.getTotalBOT()} BOT
```
Comment (dwalintukan) [inline src/components/NavBar/index.js]: use string template

File: src/components/NavBar/index.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    return (
+      <Button className={classes.navBarRightButton}>
+        <FormattedMessage id="navBar.activities" />
+      </Button>
+    );
+  }
+
+  getTotalQTUM() {
+    return _.sumBy(this.props.walletAddrs, (wallet) => wallet.qtum);
```
Comment (dwalintukan) [inline src/components/NavBar/index.js]: lets fix this to 2 decimal places

File: src/components/NavBar/index.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  }
+
+  getTotalBOT() {
+    return _.sumBy(this.props.walletAddrs, (wallet) => wallet.bot);
```
Comment (dwalintukan) [inline src/components/NavBar/index.js]: lets fix this to 2 decimal places

**Pull request merged**